packages/nodes-base/nodes/ConvertKit/FormDescription.ts

```diff
@@ -0,0 +1,174 @@
+import {
+	INodeProperties
+} from 'n8n-workflow';
+
+export const formOperations = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'form',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Add Subscriber',
+				value: 'addSubscriber',
+				description: 'Add a subscriber.',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Get a list of all the forms for your account.',
+			},
+			{
+				name: 'Get Subscriptions',
+				value: 'getSubscriptions',
+				description: 'List subscriptions to a form including subscriber data.',
+			},
+		],
+		default: 'addSubscriber',
+		description: 'The operations to perform.',
+	},
+] as INodeProperties[];
+
+export const formFields = [
+	{
+		displayName: 'Email Address',
+		name: 'email',
+		type: 'string',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'form',
+				],
+				operation: [
+					'addSubscriber',
+				],
+			},
+		},
+		default: '',
+		description: 'Subscriber email address.',
```

Comment:
To be consistent with other descriptions, maybe `The subscriber's email address.`.